docs/README.md

```diff
@@ -6,3 +6,4 @@ This repository contains documention for AMWA NMOS EDID Connection Management.
 
 - [README.md](README.md) -- This file.
 - [1.0 Architecture.md](1.0.%20Architecture.md) -- Architecture for NMOS EDID Connection Management.
+- [AMWA BCP-XXX-YY.md](1.0.%20EDID-Mapping.md) -- Mapping of EDID Information to NMOS Resources.
```

Comment:
@peterbrightwell Could on repository define an IS and a BCP?

Reply:
I think we'd rather not. One repo is one spec. (One spec repo may define more than one API obviously.)

Reply:
Ok - So looks like we need a separate repo for the BCP?  Can we move ahead using the repo we have and adapt later or best to set that up first?